windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Get unused driver letter"
+  include_tasks: ../utils/win_get_unused_drive_letter.yml
+
+- name: "Initialize the msu file path"
+  ansible.builtin.set_fact:
+    msu_file_src_path: "{{ drive_letter_new }}:\\{{ windows_update_msu_path }}\\*"
+    msu_file_dest_path: "C:\\msu"
+    msu_file_name: ""
+
+- name: "Check if folder {{ msu_file_dest_path }} exists on guest OS"
+  include_tasks: ../utils/win_is_folder.yml
+  vars:
+    win_is_folder_path: "{{ msu_file_dest_path }}"
+
+- name: "Create folder {{ msu_file_dest_path }} on guest OS"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "mkdir {{ msu_file_dest_path }}"
+  when: not win_is_folder_result
```

Comment:
What if there is a file named `msu` under C: drive? Will this task succeed?

Reply:
Yes. The dir can be created.